python/test/lib/libvmaf_libtest.py

```diff
@@ -8,13 +8,15 @@
 __copyright__ = "Copyright 2016-2018, Netflix, Inc."
 __license__ = "Apache, Version 2.0"
 
-class TestLibRunner(VmafossExecQualityRunner):
+
+class LibRunner(VmafossExecQualityRunner):
```

Comment:
Renamed to `LibRunner` because otherwise `TestLibRunner` issues a warning that this is not a valid test case (which it isn't supposed to be... it's just that class prefix `Test` seems to imply that it is by convention...)